{"name": "test_003", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 is '1'", "trace": "E   AssertionError: assert 1 is '1'"}, "description": "第三个测试用例", "start": 1598415561031, "stop": 1598415561031, "uuid": "cb736273-c908-4dc4-ad95-76cbe477e66e", "historyId": "50ba7917f2f9dab2e67b45b18c973a80", "testCaseId": "0ed793324da7e527af08def7e9f666c6", "fullName": "test_login_html.TestLogin#test_003", "labels": [{"name": "suite", "value": "test_login_html"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "LAPTOP-O3VG549U"}, {"name": "thread", "value": "3632-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_html"}]}